packages/boot/test/bootstrapTests/test-vaults-integration.ts

```diff
@@ -136,6 +137,8 @@ test('adjust balances', async t => {
   });
 });
 
+// This test isn't marked .serial, but it depends on previous tests.
```

Comment:
how can it pass reliably running in parallel with the test it depends on?

Reply:
I think this is brittle but solving it is out of scope. Filed as https://github.com/Agoric/agoric-sdk/issues/8613